libcaf_net/caf/net/dsl/client_config.hpp

```diff
@@ -39,6 +39,18 @@ struct server_address {
   uint16_t port;
 };
 
+/// Configures the integer type used as the length prefix.
+enum class size_field_t {
```

Comment:
We use the `_t` suffix mainly for STL-style type traits, not for proper class/enum names.

```suggestion
enum class size_field_type {
```